analytics-test.rb

```diff
@@ -0,0 +1,29 @@
+require 'FileUtils'
+
+puts "Enter the filetype you want to search (rb, erb, md, or html)"
+filetype = gets.chomp.to_s
```

Comment:
```
file_type = nil
while !%w( rd erb md html ).include?(file_type)
  puts "Enter the filetype you want to search (rb, erb, md, or html)"
  file_type = gets.chomp.to_s
end
```

will keep looping until you get an actual proper value. Also, use `file_type` instead of `filetype` - standard ruby style